samza-sql/src/test/java/org/apache/samza/sql/e2e/TestSamzaSqlTable.java

```diff
@@ -58,7 +58,7 @@ public void testEndToEndWithKey() throws Exception {
     TestIOResolverFactory.TestTable.records.clear();
     Map<String, String> staticConfigs = SamzaSqlTestConfig.fetchStaticConfigsWithFactories(numMessages);
 
-    String sql1 = "Insert into testDb.testTable.`$table` select id __key__, name from testavro.SIMPLE1";
+    String sql1 = "Insert into testDb.testTable.`$table`(id,name) select id __key__, name from testavro.SIMPLE1";
```

Comment:
After this checkin, Do we expect users to always specify the field names in insert into?

Reply:
No, this typically happens only in tests where only a subset of the output schema fields are projected in select statement. In real world scenario, the output schema should reflect the fields projected in the select statement.